app/screens/ScheduleScreen/ScheduleDayPicker.tsx

```diff
@@ -1,44 +1,126 @@
 import { observer } from "mobx-react-lite"
 import React, { FC } from "react"
 import { Dimensions, Pressable, TextStyle, View, ViewStyle } from "react-native"
-import Animated, { useAnimatedStyle, useSharedValue, withTiming } from "react-native-reanimated"
-import { Text } from "../../components"
+import Animated, {
+  useAnimatedStyle,
+  SharedValue,
+  interpolate,
+  interpolateColor,
+} from "react-native-reanimated"
 import { useStores } from "../../models"
-import { colors, spacing } from "../../theme"
+import { colors, spacing, typography } from "../../theme"
+import { reportCrash } from "../../utils/crashReporting"
 import { formatDate } from "../../utils/formatDate"
 
-export const ScheduleDayPicker: FC = observer(function ScheduleDayPicker() {
+type Props = {
+  scrollX: SharedValue<number>
+  onItemPress: (itemIndex) => void
+}
+
+const { width } = Dimensions.get("window")
+
+export const ScheduleDayPicker: FC<Props> = observer(function ScheduleDayPicker({
+  scrollX,
+  onItemPress,
+}) {
   const { schedulesStore } = useStores()
   const { setSelectedSchedule, schedules, selectedSchedule } = schedulesStore
-  const leftValue = useSharedValue(0)
-  const wrapperWidth = Dimensions.get("screen").width - spacing.extraSmall * 2
+  const wrapperWidth = width - spacing.extraSmall * 2
   const widthSize = wrapperWidth / schedules.length
+  const index = schedules.findIndex((s) => s.date === selectedSchedule.date)
+
+  const containerRef = React.useRef()
+  const [measures, setMeasures] = React.useState([{ x: 0 }, { x: 0 }, { x: 0 }])
+  const itemRefs = schedules.map((_) => React.createRef<View>())
+
+  const onLayout = React.useCallback(() => {
+    const m = []
+    schedules.every((_, index) => {
+      itemRefs[index].current.measureLayout(
+        containerRef.current,
+        (x, y, width, height) => {
+          m.push({ x, y, width, height })
+
+          if (m.length === schedules.length) {
+            setMeasures(m)
+          }
+        },
+        () => {
+          reportCrash("ScheduleDayPicker-unable to measureLayout")
+        },
+      )
+      return true
+    })
+  }, [itemRefs])
+
+  const inputRange = schedules.map((_, index) => index * width)
```

Comment:
The input range is made up of 3 values, and since we're interpolating off of the scrolling X value from the horizontal FlatList, this array is made up of where the width breaks are (min being 0 and start of the 3rd position 2 screen widths).